tracim/tracim/lib/notifications.py

```diff
@@ -285,7 +294,13 @@ def notify_content_update(self, event_actor_id: int, event_content_id: int):
             message['Subject'] = subject
             message['From'] = self._get_sender(user)
             message['To'] = to_addr
-
+            message['Reply-to'] = formataddr(('',replyto_addr))
```

Comment:
The good way to reply would be for example to reply to "Damien Accorsi dans l'espace de travail "Intranet" <some@email.address>